### PR TITLE
[stable/rabbitmq] Add global StorageClass to rabbitmq

### PR DESCRIPTION
**Pull request by miguelaeh**
Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by rafariossaa** [comment]
LGTM.
Could you @juan131 merge it?
/assign @juan131

File: stable/rabbitmq/templates/_helpers.tpl [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (juan131) [inline stable/rabbitmq/templates/_helpers.tpl]: Remove extra line

**Review by juan131 — COMMENTED** [review]
Thanks for the PR!

**Review by juan131 — APPROVED** [review]

**Pull request merged**